pokemongo_bot/api_wrapper.py

```diff
@@ -61,13 +61,16 @@ def call(self, max_retry=5):
             self._api._req_method_list = [req_method for req_method in api_req_method_list] # api internally clear this field after a call
             result = self._api.call()
             if not self._is_response_valid(result, request_callers):
-                try_cnt += 1
-                logger.log('Server seems to be busy or offline - try again - {}/{}'.format(try_cnt, max_retry), 'red')
-                if try_cnt >= max_retry:
-                    raise ServerBusyOrOfflineException()
-                sleep(1)
-            else:
-                break
+                while try_cnt < max_retry:
+                    try_cnt += 1
```

Comment:
I don't get what you are trying to do here. You are just incrementing and sleeping. Nothing else happens. Correct me if I'm wrong